quick_start.rst

```diff
@@ -190,7 +190,7 @@ Then you will be able to check installed Behat version using:
 
 .. code-block:: bash
 
-    $ vendor/bin/behat -V
+    $ php vendor/behat/behat/bin/behat -V
```

Comment:
I suppose it should be `php vendor/bin/behat -V`

Reply:
Sorry, don't know :-( Do you want me to change it?

Reply:
I don't have merge access to this repo, but the path should be `vendor/bin/behat` so someone could merge it